t/stream/ssl.t

```diff
@@ -0,0 +1,1826 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua::Stream;use Cwd qw(cwd);
```

Comment:
Is it intentional that there isn't a new line between those two statements? The `t/ssl.t` test suite has such a new line.

Reply:
No that was definitely not intentional. Fixed.